README_es.md

```diff
@@ -0,0 +1,23 @@
+# NetBeans Encoding Plugin
+Este plugin abre/guarda un archivo con una codificación especificada
+
+## Requirements
+
+- NetBeans7.4+
+## Open/Save in Encoding
+File > Open/Save in Encoding...
+
+La codificación actual se mostrará en la Barra de Herramientas. Si desea cambiar la codificación, por favor seleccione la codificación específica.
+( Nota:Entonces, el archivo se cierra y se vuelve a abrir.)
+
+Si no desea que se muestre deseleccione la casilla `View > Toolbar > Encoding`.
+
+## License
+[Common Development and Distribution License (CDDL) v1.0 and GNU General Public License (GPL) v2](http://netbeans.org/cddl-gplv2.html)
+
+ ----
+Cloned from encoding module of main/contrib repository. (Tomas Zezula)
+Also see https://netbeans.org/bugzilla/show_bug.cgi?id=114123
+
+Translator : @javatlacati
+Contact: scherzo16@gmail.com
```

Comment:
Could you please change the following?

Translator : @javatlacati (scherzo16@gmail.com)